### PR TITLE
User can view a timeline.

### PR DESCRIPTION
**Pull request by aidanbdh**

### CODE REVIEW

File: default.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#timeline {
+  display: inline-block;
+  width: 65vw;
+  float: right;
+}
+
+#post {
+  display: inline-block;
+  margin: auto;
+  float: center;
```
Comment (ronperris) [inline default.css:10]: Consider this part. Do you need it?

File: default.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+#post textarea {
+  resize: none;
+}
+
+#post button {
+  float:right;
```
Comment (ronperris) [inline default.css:18]: Missing a space after the colon.

File: default.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+.post {
+  position: relative;
+  height: 136px;
+  width: 601px;
+  padding: 10px;
+  padding-bottom: 20px;
+  margin-top: 30px;
+  border-style: solid;
+  border-color: rgba(0,0,0,1);
```
Comment (ronperris) [inline default.css:29]: Why does this have an opacity?

File: default.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+#post textarea {
+  resize: none;
+}
+
+#post button {
+  float:right;
+}
+
+.post {
+  position: relative;
```
Comment (ronperris) [inline default.css:22]: Why is this position relative?

File: timeline.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Facebook</title>
+    <link rel="stylesheet" href="default.css">
+  </head>
+  <body>
+    <div id="timeline">
+      <form id="post">
+        <textarea required = "true" cols="100" rows="10" minlength="1" resize="none" id="posttext">Text Here</textarea>
```
Comment (ronperris) [inline timeline.html:10]: Remove spaces between key/value and equal signs.

Comment (ronperris) [reply timeline.html:10]: Remove resize from here or CSS.

File: timeline.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+        <div>
+          <button type="submit" name="submit">Post</button>
+        </div>
+      </form>
+      <div></div>
```
Comment (ronperris) [inline timeline.html:15]: Remove this div.

File: timeline.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
+console.log('We are here, we are here, we are here!')
```
Comment (ronperris) [inline timeline.js:1]: Remove the console.log.

File: timeline.js [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+var timeline = function(){
```
Comment (ronperris) [inline timeline.js:2]: Add a space.

File: timeline.js [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+    "3Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean mollis ornare lacus, id maximus est hendrerit nec. Cras vel congue nunc, eget lobortis nisi. Suspendisse non erat neque. Nulla luctus ut velit vestibulum suscipit. Cras quis ultrices diam. Fusce semper diam vel odio euismod facilisis. Vivamus quis dignissim tortor. Nam viverra libero id enim tincidunt aliquam. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas maximus metus at mi ullamcorper, ac posuere ipsum sodales. Maecenas malesuada faucibus enim sit amet feugiat. Etiam elit nisi, accumsan et tempus ut, auctor sit amet nisl."
+  ];
+  //Default textbox text
```
Comment (ronperris) [inline timeline.js:11]: Consider if the event listeners should be added inside the function call.

File: timeline.js [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+  formtext.addEventListener('click', function(e) {
+    if(formtext.value === formtext.defaultValue) {
```
Comment (ronperris) [inline timeline.js:13]: Add a space between keyword and condition.

File: default.css (hunk 1 above)
Comment (aidanbdh) [reply default.css:10]: I do not

File: default.css (hunk 2 above)
Comment (aidanbdh) [reply default.css:18]: float: right;

**Pull request merged**